File: tdd-examples/ruby/fizzbuzz.rb
def fizzbuzz(number)
  return 'FizzBuzz' if ! number.zero? && (number % 3).zero? && (number % 5).zero?
  return 'Buzz' if ! number.zero? && (number % 5).zero?
  return 'Fizz' if ! number.zero? && (number % 3).zero?

  number
end

File: tdd-examples/node/fizzbuzz.js
function fizzbuzz(n) {
  if(n!=0 && n%5==0 && n%3==0)
    return 'FizzBuzz';
  if(n!=0 && n%3==0)
    return 'Fizz';
  if(n!=0 && n%5==0)
    return 'Buzz';
  return n;
}

module.exports = fizzbuzz
